pkg/apis/serving/v1alpha1/configuration_validation.go

```diff
@@ -52,6 +52,7 @@ func (cs *ConfigurationSpec) Validate(ctx context.Context) *apis.FieldError {
 
 	errs := CheckDeprecated(ctx, map[string]interface{}{
 		"generation": cs.DeprecatedGeneration,
+		// TODO(mattmoor): "revisionTemplate": cs.RevisionTemplate,
```

Comment:
Can you update these with an issue number instead?

Reply:
Technically Github PRs are issues (in their API), so I'll use #3816, which is next up 😅